modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -42,6 +42,8 @@ Exchange prevents resource exhaustion attacks by limiting the number of asset ve
 
 The limit is 500 assets for root organizations with trial accounts, and 100,000 (one hundred thousand) assets for other root organizations.
 
+Exchange limits assets to 100 dependencies. Additionally, Anypoint API Designer limits RAMLs to 100 RAML Fragments as dependencies, and Anypoint API Manager limits API Groups to 100 REST APIs as dependencies.
```

Comment:
```suggestion
Exchange limits assets to 100 dependencies. Additionally, that means Anypoint API Designer limits RAMLs to 100 RAML Fragments as dependencies, and Anypoint API Manager limits API Groups to 100 REST APIs as dependencies.
```

Reply:
what about adding something like that? Since `Exchange limits assets to 100 dependencies` implies that de limit is 100 for RAML Fragments and API Groups, I would just add the second part as a reminder

Reply:
Another possibility: "The limit of dependencies for an asset is 100. For example, in Anypoint API Designer a RAML has up to 100 RAML Fragment dependencies, and in Anypoint API Manager an API Group has up to 100 REST API dependencies."